components/canvas/Canvas.tsx

```diff
@@ -24,3 +20,26 @@ function Canvas() {
   );
 }
 export default Canvas;
+
+const albert = { x: 1360, y: 250, width: 50, height: 10 };
+
+function moveAlbert(context) {
+  const image = new Image();
+  image.src = "/Albert.svg";
+  image.onload = () => {
+    context.drawImage(image, albert.x, albert.y);
+  };
+  context.clearRect(0, 0, albert.width, albert.height);
+  if (albert.x == albert.y) {
+    albert.y += 10;
+  } else if (albert.x === albert.y) {
+    albert.y -= 10;
+  } else if (albert.y === albert.x) {
+    albert.x += 10;
```

Comment:
Der Effekt dieses Codes wird mir noch nicht ganz klar. Gibt es einen Grund, wieso albert.x nicht albert.y sein darf? ``albert.x == albert.y`` wäre immer true, wenn sich Albert auf der Diagonalen von der linken oberen Ecke bis zur rechten unteren Ecke befinden würde. 

Zudem überprüft in meinen Augen ``albert.x === albert.y`` und ``albert.y === albert.x`` dasselbe. Da könntest du also ``else if (albert.x === albert.y) {   albert.y -= 10;   albert.x += 10;  }  `` schreiben.

Reply:
```suggestion
  if (albert.x === albert.y) {
    albert.y += 10;
  } else if (albert.x === albert.y) {
    albert.y -= 10;
  } else if (albert.y === albert.x) {
    albert.x += 10;
```
Like Josca wrote ... in the end you are comparing three times the same values? 🙃 
maybe you need to compare with > or < 

maybe look at this to avoid confusing if-else Statements: 
https://www.youtube.com/watch?v=EumXak7TyQ0